### DISCUSSION TITLE
Compile bundled JavaScript into bytecode

### DISCUSSION THREAD

**Discussion by kwaitsing** [oven-sh/bun#9507] [Ideas]
Probably to gain a better performance?

**Comment by 7f8ddd** [comment]
That would be cool, similar to Lua.

**Comment by pavi2410** [comment]
Bun could compile to bytecode like [Hermes](https://github.com/facebook/hermes)

**Comment by ouarrho** [comment]
Great Idea, also get executable files for mobile, desktop, IoT etc...

**Comment by nexusprime-here** [comment]
Looks great, pls make it bun 🙏
